docs/pages/management/dynamic-resources/terraform-provider.mdx

```diff
@@ -30,15 +30,13 @@ and Machine ID](../../machine-id/deployment/spacelift.mdx).
 ## Step 1/3. Create Teleport credentials for Terraform
 
 Terraform needs a signed identity file from the Teleport cluster certificate
-authority to manage resources in the cluster.
-You can create a local Teleport user for this purpose or you can use the machine identity agent
-[(Machine ID)](../../machine-id/introduction.mdx) to generate credentials.
+authority to manage resources in the cluster. Terraform is intended to be used with
+[(Machine ID)](../../machine-id/introduction.mdx), so this tutorial will show you how to
+use a local Machine ID bot to try it out.
 
-If you intend to run Terraform from a CI/CD platform, Machine ID is often a better
-option for generating credentials. Machine ID can provision ephemeral short-lived
-certificates that are appropriate for CI/CD workflows instead of using manually-generated
-credentials that have a longer time-to-live (TTL) period. For more information about
-using Machine ID, see the [Machine ID Getting Started Guide](../../machine-id/getting-started.mdx).
+For production, Terraform should be run using Machine ID on a CI/CD platform.
+For more information on using Machine ID and Terraform in a production setting, see the
+[Machine ID Terraform Guide](../../machine-id/access-guides/terraform.mdx).
```

Comment:
I would move this to "Next steps" so the reader can sprint through the tutorial first before planning work for production.

Reply:
Updated!